src/components/header/Header.component.tsx

```diff
@@ -1,2 +1,46 @@
-import styles from './header.module.css';
-export const HeaderComponent = () => <header className={styles.header}>Header</header>;
+import maLogo from '@/assets/ma.svg';
+
+import './header.css';
+
+export const HeaderComponent = () => (
+    <section className="head">
+        <div className="head__logo">
+            <a href="https://www.mastersacademy.education/" target="_blank" rel="noreferrer" className="head__logo-ma">
+                <img src={maLogo} alt="Masters Academy" className="head__logo-ma--link" />
+            </a>
+        </div>
+
+        <div className="head__theme">
+            <button className="head__theme--sun active">
```

Comment:
Create reusable Button component, which can accept all native button props and reuse, it
Need to use same pattern for reusable UI components as for Link
https://github.com/Grimdvl/fe-react-2024/pull/1/files#r1575022620